consensus/src/consensus.rs

```diff
@@ -325,6 +325,9 @@ where
                         .consensus_dag_rounds
                         .with_label_values(&[])
                         .set(state.dag.len() as i64);
+                    self.metrics
+                        .dag_size_bytes
+                        .set((mysten_util_mem::malloc_size(&state.dag) + std::mem::size_of::<Dag>()) as i64);
```

Comment:
What is the order of magnitude of the error we expect from the second argument of the sum?

Reply:
Probably quite small, but this is a fixed value so I'd hope that the call to size_of is inlined as a constant? Not sure how to check that though.

Reply:
We actually came up on this a while ago: https://github.com/MystenLabs/sui/pull/3920#discussion_r943290604

Reply:
In what way do you mean you came up on it? Clearly the heap allocated part needs to be measured (which is what that discussion seemed to be about), but what's the harm in also adding the stack allocated part to the measurement? Is there a nuance I'm missing here?

Reply:
Sorry about that, my comment above was misguided and not very constructive, I was more thinking out loud, and that led you on a wild goose chase, which I apologize for.